### ISSUE TITLE
Translation of LightBulb.md is having issues. 

### ISSUE THREAD

**Issue by greg-norris** [microsoft/pxt-brainpad#119]
For some reason the words inside the "New, Try this First - LightBulb" tutorial (lightbulb.md) aren't translating correctly. It is the only one that apparently has an issue. It works fine in English. The Title and Inline code snippets are translating correctly, just the instruction in the tutorial are not as shown in the image below. 

![translationissue](https://user-images.githubusercontent.com/29868099/47174008-870a3a80-d2dd-11e8-83e6-a2397cdc3dbf.jpg)

This is how we would expect it to look as seen in another tutorials shown below. 

![translationissue2](https://user-images.githubusercontent.com/29868099/47174154-d9e3f200-d2dd-11e8-840a-e9ed0bc50cc6.jpg)

**Comment by pelikhan** [comment]
Missing newlines after the ## titles confuses crowdin.

**Comment by pelikhan** [comment]
I've bump an update to the markdown please try again.

**Comment by greg-norris** [comment]
@pelikhan Hmmm...all the other tutorials are missing a line space after ## and work correctly. Now the Lightbulb tut, won't load at all.

**Comment by pelikhan** [comment]
Yes because crowdin seems to have a parking issue and goggles the paragraph in the ## header. You will have to reapply translations afterwards.

**Comment by pelikhan** [comment]
(Bump to upload the translations)

**Comment by greg-norris** [comment]
Awesome, thanks. Any reason why the name of the lightbulb.md was giving me issue vs light.md?

**Comment by pelikhan** [comment]
Might be caching. Try an anonymous window. Maybe see in crowdin to see if something is obviously broken.

**Comment by greg-norris** [comment]
@pelikhan Okay, here's the latest I found and its kind of a peculiar issue. I fixed the Lightbulb tutorial by changing its name to light.md, everything worked fine. Then I bumped and tagged the version to 0.21.16. After I did this the lightbulb tutorial would no longer load again. To verify this I changed the name from light.md to led.md. Cleared all the caching, works again, as soon as I bump to v0.21.17 led.md no longer works.
